src/Microsoft.AspNetCore.Server.Kestrel/Infrastructure/TaskUtilities.cs

```diff
@@ -26,12 +26,15 @@ public static Task GetCancelledTask(CancellationToken cancellationToken)
 #endif
         }
 
-        public static Task<int> GetCancelledZeroTask()
+        public static Task<int> GetCancelledZeroTask(CancellationToken cancellationToken)
```

Comment:
I would add an overload `GetCancelledZeroTask()` which calls `GetCancelledZeroTask(CancellationToken.None)`.  This is cleaner than requiring the caller to pass `default(CancellationToken)` or `CancellationToken.None`.